components/dashboard/component/ColoredCard.js

```diff
@@ -1,15 +1,22 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import classNames from 'classnames';
 
-const ColoredCard = ({ backgroundColor, icon, title, total }) => {
+const ColoredCard = ({ backgroundColor, fourColumn, icon, title, threeColumn, total }) => {
```

Comment:
```suggestion
const ColoredCard = ({ backgroundColor, fourColumn, icon, title, threeColumn, total }) => {
```
```suggestion
const ColoredCard = ({ backgroundColor, isThreeColumn, icon, title, total }) => {
```

As discussed, we can use boolean `isThreeColumn` or allow the user to set the number of column by passing directly.